formats/json/commonMain/src/kotlinx/serialization/json/internal/Polymorphic.kt

```diff
@@ -16,7 +16,7 @@ import kotlin.jvm.*
 internal inline fun <T> JsonEncoder.encodePolymorphically(
     serializer: SerializationStrategy<T>,
     value: T,
-    ifPolymorphic: (String) -> Unit
+    ifPolymorphic: (String, String) -> Unit
```

Comment:
[nit] It will be easier to understand if function type had parameter names:

```suggestion
    ifPolymorphic: (discriminatorName: String, serialName: String) -> Unit
```